### PR TITLE
ODQA model

### PR DESCRIPTION
**Pull request by my-master**
This pull-request includes the following main changes:
* ODQA test config (infer only)
* ODQA prod config (infer only)
* odqa moved to skills
* wiki_sqlite vocab (a connection component between ranker and reader)
* ODQA test config included in tests

**Update**
* remove start answer position from the final ODQA output in prod config
* enable batch processing in WikiSqlite vocab

### CODE REVIEW

File: deeppavlov/vocabs/wiki_sqlite.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    """
+
+    def __init__(self, data_dir: str = '', data_url: str = DB_URL, **kwargs):
+        """
+        :param data_dir: a directory name where DB is located
+        :param data_url: an URL to SQLite DB
+        """
+        download_dir = expand_path(data_dir)
+        download_path = download_dir.joinpath(data_url.split("/")[-1])
+        download(download_path, data_url, file_exists=False)
```
Comment (yoptar) [inline deeppavlov/vocabs/wiki_sqlite.py]: Can we rename `file_exists` to something like `force_download` or `redownload`?

File: deeppavlov/core/data/utils.py [hunk 2]
```diff
@@ -91,7 +91,7 @@ def ungzip(file_path, extract_folder=None):
             extract_folder: folder to which the files will be extracted
 
         """
-    CHUNK = 16 * 1024
+    CHUNK = 2 ** 14
```
Comment (yoptar) [inline deeppavlov/core/data/utils.py]: I think that 16 * 1024 is more readable

Comment (my-master) [reply deeppavlov/core/data/utils.py]: Done

File: deeppavlov/vocabs/wiki_sqlite.py (hunk 1 above)
Comment (my-master) [reply deeppavlov/vocabs/wiki_sqlite.py]: Done

**Pull request merged**